### PR TITLE
Add branding 1

### PR DESCRIPTION
**Pull request by Aichnerc**
**Checklist:**

 
 - [x] Have you added an explanation of what your changes do and why you'd like them to be included?
 - [ ] Have you updated or added documentation for the change?
 - [x] Have you tested your changes with successful results?

**Type of Changes:**

 
 - [ ] Bug fix (non-breaking change which fixes an issue)
 - [x] New feature (non-breaking change which adds functionality)
 - [ ] Breaking change (fix or feature that would cause existing functionality to change)

**What is the current behavior? (link to any open issues here)**
- Currently, there is no branding page. #32 

**What is the new behavior (if this is a feature change)?**
- A branding page has been added. #32 

**Other information**:
- The logo images can not be downloaded yet.
- The version has been updated to v0.2.0.

[Learn more about our versioning](https://github.com/aichner/Company-Homepage#versioning)

### CODE REVIEW

**Review by Aichnerc — COMMENTED** [review]
It looks clean to me!

**Review by sweggerat — APPROVED** [review]
Looks good.

File: src/components/pages/BrandingPage/index.jsx [hunk 1]
```diff
@@ -0,0 +1,367 @@
...
+  MDBCol,
+  MDBCard,
+  MDBCardBody,
+  MDBTable,
+  MDBTableBody,
+  MDBBtn,
+  MDBIcon,
+} from 'mdbreact';
+
+//> CSS
```
Comment (kleberbaum) [inline src/components/pages/BrandingPage/index.jsx:19]: SCSS?

There is a lot of inconsistency with CSS/SCSS headline comments.
image

**Review by kleberbaum — CHANGES_REQUESTED** [review]
Should be ok to merge after a few changes.

**Review by kleberbaum — APPROVED** [review]
Should be ok to merge.

**Pull request merged**